commit cc72020f11d51c7482abca4f98daf76e02ef7105
Author: ccobine <carol.cobine@joyent.com>
Date:   2019-03-15T15:38:11-04:00 (7 months ago)
    
    DOC-648 Please add that MD5 headers require base64 when adding a header for upload (fix markdown syntax)

diff --git a/docs/user-guide/storage-reference.md b/docs/user-guide/storage-reference.md
index 1691c40..f7124f6 100644
--- a/docs/user-guide/storage-reference.md
+++ b/docs/user-guide/storage-reference.md
@@ -1,4 +1,4 @@
- ---
+---
 title: Object Storage Reference
 markdown2extras: wiki-tables, code-friendly
 ---
